app/lib/experience.js

```diff
@@ -0,0 +1,124 @@
+//functions intereacting with player experience points in Minecraft will go here.
+module.exports = Base => class extends Base {
+    constructor() {
+        super();
+    }
+
+    //minecraft seperates levels and points you need to get both.
+    readPlayerExperiencePoints(playerName) {
```

Comment:
I've been trying to reserve the words create, read, update, delete for functions that actually deal with the database. Since this doesn't get into the database could you think of a different name for `readPlayerExperienceLevels` and `readPlayerExperiencePoints`